### ISSUE TITLE
Use as a library

### ISSUE THREAD

**Issue by wcauchois** [Ephigenia/ikea-availability-checker#17]
It doesn't seem like this code is usable as a library. Would you consider adding the ability for this to be used as a library, as well as a CLI program?

**Comment by Ephigenia** [comment]
Sure

**Comment by Ephigenia** [comment]
The first issue identified seems to be that the `"main"` entry in the `package.json` points to the `source/cli.js` script which becomes the file that is loaded when `require('ikea-availability-checker');` is loaded: https://github.com/Ephigenia/ikea-availability-checker/blob/master/package.json#L39

I’ll have to check what’s the most common cause to solve this. Sure there’ll be a solution.

**Comment by Ephigenia** [comment]
I’ve been thinking about this. The most anticipated feature I can think of would be "What’s the current item in Store B of product A" … which could look like this:

```javascript
// buCode is country specific
const checker = require('ikea-availability-checker');
await checker.availability(buCode, productId);
```

What bothers me is the question of how the user/programmer knows about the buCode. There are several lookup functions in the [stores](https://github.com/Ephigenia/ikea-availability-checker/blob/master/source/lib/stores.js) but they might be to difficult to use.

@wcauchois can you give me some insights on how you'd like to use the lib?

**Comment by Ephigenia** [comment]
@wcauchois I’ve published a new version v0.5.0 which you now can use in your library. See the example in the README.md: https://github.com/Ephigenia/ikea-availability-checker#api

**Comment by wcauchois** [comment]
That's great! Thank you @Ephigenia.
